EditPortfolio.css

```diff
@@ -0,0 +1,369 @@
+  height: 100%;
+  float: right;
+  margin-top: 40px;
+  right: 1px;
+  display: flex;
+  flex-direction: column;
+}
+.sidebar li {
+  font-family: "Fjalla One", sans-serif;
+  font-size: 16px;
+  color: #5d5151;
+  display: flex;
+}
+.borderSidebar {
+  width: 100%;
+  height: 88%;
+  margin-top: 30px;
+  border: 5px solid #ffffff;
+  border-width: 2px;
+  border-radius: 5px;
+  margin-bottom: 10px;
+}
+.skillsContainer {
+  margin-top: 50px;
+  width: 100%;
+  padding-top: 5px;
+  height: auto;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.introduction_text {
+  font-family: "Francois One", sans-serif;
+  font-size: 22px;
+  margin-bottom: 20px;
+  color: #555;
+  margin-top: 5px;
+  margin-bottom: 10px;
+  text-align: center;
+}
+.avatar {
+  margin-top: 50px;
+  width: 100%;
+  border-radius: 50%;
+  border: 5px solid #ffffff;
+  height: auto;
+  margin: 0 auto;
+}
+.imgNameFont {
+  font-family: "Fjalla One", sans-serif;
+  font-size: 22px;
+  color: #000000;
+  margin-top: 5px;
+  margin-bottom: 10px;
+}
+.skillsHeading {
+  text-decoration: underline;
+  font-family: "Francois One";
+  font-size: 20px;
+  color: #000000;
+  margin-top: 35px;
+  margin-bottom: 3px;
+}
+
+.skills {
+  font-family: "Fjalla One", sans-serif;
+  font-size: 18px;
+  color: #5d5151;
+  line-height: 15px;
+  margin-top: 10px;
+  align-self: center;
+}
+
+#container {
+  width: 75%;
+  padding: 20px;
+  margin-left: 10px;
+}
+.border {
+  border: 5px solid #ffffff;
+  border-width: 2px;
+  border-radius: 5px;
+  padding-top: 0px;
+  padding-left: 15px;
+  padding-right: 5px;
+  padding-bottom: 3px;
+  margin-bottom: 10px;
+}
+
+.title {
+  margin-top: 50px;
+
+  font-family: "Fjalla One", sans-serif;
+  font-size: 30px;
+  color: #000000;
+  margin-bottom: 10px;
+}
+.heading2 {
+  font-family: "Francois One";
+  font-size: 22px;
+  color: #555;
+  margin-top: 10px;
+  margin-bottom: 3px;
+}
+
+.paragraph {
+  color: #5d5151;
+  margin-top: 5px;
+  line-height: inherit;
+  justify-content: right;
+  padding-right: 20px;
+  text-align: justify;
+  font-size: 16px;
+}
+
+.linkedIn {
+  color: #5d5151;
+  margin-top: 5px;
+  font-size: 15px;
+  font-weight: bold;
+}
+.linkedIn a {
+  margin-left: 30px;
+  text-decoration: none;
+  color: #ffffff;
+}
+.linkedIn a :hover {
+  color: #d6ebba;
+}
+
+footer {
+  background-color: #333;
+  color: #fff;
+  padding: 10px 0;
+  width: 100%;
+}
+
+.addButton {
+  background-color: #17a2b8;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  padding: 10px 20px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+  margin-top: 0px;
+  margin-bottom: 10px;
+}
+.span {
+  color: rgb(38, 38, 39);
+  font-weight: bold;
+  margin-right: 10px;
+}
+
+.model {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+.model-content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 50%;
+  margin-top: 40px;
+  height: auto;
```

Comment:
You can use only top to adjust the position. No need to use margin top here since it is already position absolute.

Reply:
noted